wazimap_ng/profile/serializers/metrics_serializer.py

```diff
@@ -53,14 +53,16 @@ def MetricsSerializer(profile, geography, version):
         method = algorithms.get(denominator, absolute_value)
         val = method(profile_key_metric, geography)
         if val is not None:
+            metadata = MetaData.objects.filter(dataset=profile_key_metric.variable.dataset).values("source", "description", "url").first()
```

Comment:
QQ, there is no way of getting metadata when fetching `profile_key_metrics` directly i.e. similar to how they do it [here](https://github.com/OpenUpSA/wazimap-ng/blob/a6353e2cb96c916b568d3e9e6721556f88373af1/wazimap_ng/profile/serializers/utils.py#L32-L34) . Trying to avoid this extra DB call we have to make here.